industrial_trajectory_filters/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 2.8.3)
 
 project(industrial_trajectory_filters)
 
+add_compile_options(-std=c++11)
```

Comment:
> I'm not sure I follow this. Being more specific is always a good thing in my opinion.

So, are you really using  `target_include_directories`? ;)

> We wouldn't be mixing options, as target_compile_options(..) would be used with all targets in the package. 

IMHO this is not the purpose of `target_compile_options`, it is meant for fine granular settings. If you want to set options for all targets anyway, `add_compile_options` should work as expected. It sets the options for all sources [in the current directory and below](https://cmake.org/cmake/help/v3.0/command/add_compile_options.html).